src/reducers/index.js

```diff
@@ -0,0 +1,52 @@
+import initialState from '../store/initialState';
+
+import { SortOptions } from '../constants';
+import {
+    OPEN_MOVIE_PAGE,
+    UPDATE_SORT_OPTION,
+    UPDATE_SEARCH_OPTION,
+    UPDATE_SEARCH_VALUE,
+    SAVE_FILM_LIST,
+} from '../constants/actions';
+
+const rootReducer = (state = initialState, action) => {
```

Comment:
better to use `import { combineReducers } from "redux";` to merge reducers together